drivers/hmis/app/jobs/hmis/candidate_pool_builder_job.rb

```diff
@@ -0,0 +1,28 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Hmis
+  class CandidatePoolBuilderJob < BaseJob
+    include NotifierConfig
+
+    def perform
+      log("Building candidate pools for #{Hmis::Ce::Opportunity.active.count} active opportunities")
+      Hmis::Ce::Match::CandidatePoolBuilder.new.perform
+
+      log("Running the CE match engine for #{Hmis::Hud::Client.hmis.count} HMIS clients")
+      clients = Hmis::Hud::Client.hmis
+      Hmis::Ce::Match::CandidatePool.all.each do |pool|
```

Comment:
If we expect candidate pool to be large in the future, a find_each might be warranted
```suggestion
      Hmis::Ce::Match::CandidatePool.all.find_each do |pool|
```